src/main/java/org/camunda/community/benchmarks/StartDiScheduler.java

```diff
@@ -0,0 +1,229 @@
+package org.camunda.community.benchmarks;
+
+import org.camunda.community.benchmarks.config.BenchmarkConfiguration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import java.time.Instant;
+
+@Component
+@ConditionalOnProperty(name = "benchmark.startDecisions", havingValue = "true", matchIfMissing = true)
+public class StartDiScheduler {
+
+    private static final Logger LOG = LoggerFactory.getLogger(StartDiScheduler.class);
+
+    @Autowired
+    private BenchmarkConfiguration config;
+
+    @Autowired
+    private StatisticsCollector stats;
+
+    @Autowired
+    private StartDiExecutor executor;
+
+    private long startTimeInMillis = System.currentTimeMillis();
+    private long diStarted = 0;
+    private long diStartedGoal = 0;
```

Comment:
Just call these instancesStarted and instancesStartedGoal so that they can be in the abstract base class